packages/service-library/src/servicelib/fastapi/celery/app_server.py

```diff
@@ -18,7 +19,14 @@ def __init__(self, app: FastAPI):
         super().__init__(app)
         self._lifespan_manager: LifespanManager | None = None
 
-    async def lifespan(self, startup_completed_event: threading.Event) -> None:
+    @property
+    def task_manager(self) -> TaskManager:
+        task_manager = self.app.state.task_manager
```

Comment:
The order in which the app state is setup is very important and here I do not see how this is guaranteed. Can you please show me offline how the workflow works?